models/reviews.js

```diff
@@ -1,19 +1,29 @@
 const db = require("../db/connection");
 
 exports.selectReview = (req) =>{
```

Comment:
Be careful with pluralisations. Here your `selectReview` function should be pluralised as you anticipate being able to select multiple reviews